profiling/src/collections/identifiable/mod.rs

```diff
@@ -75,9 +75,9 @@ impl<T: Item> Dedup<T> for FxIndexSet<T> {
     }
 }
 
-pub fn to_pprof_vec<T: PprofItem>(collection: &FxIndexSet<T>) -> Vec<T::PprofMessage> {
+pub fn to_pprof_vec<T: PprofItem>(collection: FxIndexSet<T>) -> Vec<T::PprofMessage> {
```

Comment:
Since it consumes, rename it `into_pprof_vec`?

Reply:
done